client/blocks/reader-export-button/api.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Internal dependencies
+ */
+import wp from 'calypso/lib/wp';
+
+export async function exportReaderList( listId ) {
+	const result = await wp.req.get( `/read/lists/${ listId }/export`, { apiNamespace: 'wpcom/v2' } );
```

Comment:
You don't need to `await` for this if it's the only thing that we're doing in this function. Instead, you can just make it a regular function and `return wp.req.get`..., and then update the consumer code to be async and `await exportReaderList( listId )`.

This actually makes the function so simple that we might as well just remove the function and directly use `wp.req.get()` in the component below.